sos/report/plugins/kdump.py

```diff
@@ -43,27 +43,31 @@ class RedHatKDump(KDump, RedHatPlugin):
     packages = ('kexec-tools',)
 
     def fstab_parse_fs(self, device):
-        with open(self.path_join('/etc/fstab'), 'r') as fp:
-            for line in fp:
+        """ Parse /etc/fstab file """
+        fstab = self.path_join('/etc/fstab')
+        with open(fstab, 'r', encoding='UTF-8') as file:
+            for line in file:
                 if line.startswith((device)):
                     return line.split()[1].rstrip('/')
         return ""
 
     def read_kdump_conffile(self):
-        fs = ""
+        """ Parse /etc/kdump file """
+        fsys = ""
         path = "/var/crash"
 
-        with open(self.path_join('/etc/kdump.conf'), 'r') as fp:
-            for line in fp:
+        kdump = '/etc/kdump'
```

Comment:
`kdump = '/etc/kdump.conf'` as it was in (redundantly called) `path_join`?

Reply:
Ack, will fix.